README.md

```diff
@@ -5,28 +5,77 @@
 
 ### What is this repository for? ###
 
-* Address Index is a Play Framework application which resolves address query strings to actual addresses.
-* Version 0.0.1
+Address Index is a Play Framework (2.6) application which matches addresses.
+
+The input can be a complete address (may be poor quality), and the system will use advanced data science techniques to determine the most likely matching AddressBase entries with UPRNs (Unique Property Reference Numbers).
+
+Addresses can be matched one at a time or in batches.
+
+Additional functions exist for postcode searching and partial address string matching for typeaheads.
+
+The application will be deployed in 2019 as a service free to all members of the Public Sector Mapping Agreement.
```

Comment:
This is a big commitment! How about:
It is planned to deploy the service in 2019 and we endeavour to make it available to members of the Public Sector Mapping Agreement.